README.md

```diff
@@ -24,10 +24,13 @@ node-lambda setup
 Ensure `deploy.env` has the following:
 ```
 DISCOVERY_STORE_CONNECTION_URI=[encrypted rds connection string]
-ELASTICSEARCH_CONNECTION_URI=[encrypted es connection string]
-NYPL_API_SCHEMA_URI=[*not* encrypted nypl data api base url]
-LOGLEVEL=info
+ELASTICSEARCH_CONNECTION_URI=[encrypted es connection string, which in plaintext could be "localhost:9200"]
 ELASTIC_RESOURCES_INDEX_NAME=[name of resources index]
+NYPL_API_SCHEMA_URL=[plaintext schema base url ending in '/current-schemas/']
+NYPL_API_BASE_URL=[plaintext data api base url ending in, for example, '/v0.1/']
```

Comment:
@nonword: I REALLY like these example values.
One thing - I think you've switched around your examples in `OUTGOING_STREAM_NAME` & `OUTGOING_SCHEMA_NAME`.